racingcar-java/src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,23 @@
+package racingcar;
+
+public class Car {
+
+    private Position position;
```

Comment:
```suggestion
    private final Position position;
```
변수 참조값을 불변해도 좋을거같네요 :)

Reply:
확인했습니다.
피드백 감사합니다 :)